teamwork/static/css/main.css

```diff
@@ -1,6 +1,43 @@
+/* Global Styles */
+
 html {
   box-sizing: border-box;
 }
 *, *:before, *:after {
   box-sizing: inherit;
 }
+
+body {
+    margin-top: 0px;
+    background-color: #fff;
+}
+
+@media(min-width:768px) {
+    body {
+        margin-top: 0px;
+    }
+}
+
+#page-wrapper {
+    width: 100%;
+    padding: 0;
+    background-color: #fff;
+}
+
+@media(min-width:768px) {
+    #wrapper {
+        padding-left: 225px;
+    }
+
+    #page-wrapper {
+        padding: 10px;
+    }
+}
+
+.jumbotron {

```

Comment:
Not that important but I think it's nice to be consistent and have same indentation here as is in the rest of the file (i.e. 4 spaces)